### PR TITLE
Update get-started.mdx

### PR DESCRIPTION
**Pull request by jeanservaas**
Few type changes:

Removed the "Create" under Shankar's contribution description. No one else had a verb.

Also under Review and Publish, I removed "to be made" to streamline the sentence.

Maintainers will be reviewing your work and making comments, asking questions and suggesting changes **to be made** before they merge your code.

### CODE REVIEW

File: src/pages/contributing/get-started.mdx [hunk 1]
```diff
@@ -66,7 +66,7 @@ Accessibility<br /><br /> **Amaya Mali**<br /> Gantt chart
 <Column className="one" colSm={2} colMd={2} colLg={3}>
 
 **Tom Roach**<br /> Community support<br /><br />**Eugene To **<br />
-Figma<br /><br /> **Shankar Venkatachalam** <br /> Create React App<br /><br />
+Figma<br /><br /> **Shankar Venkatachalam** <br /> React App<br /><br />
```
Comment (alisonjoseph) [inline src/pages/contributing/get-started.mdx:69]: "Create React App" is the correct name I believe https://create-react-app.dev/

Comment (alisonjoseph) [inline src/pages/contributing/get-started.mdx:69]: "Create React App" is the correct name I believe https://create-react-app.dev/ (not sure where the Carbon version is)

**Review by alisonjoseph — CHANGES_REQUESTED** [review]

**Review by guidari — APPROVED** [review]
LGTM! 🚀

File: src/pages/contributing/get-started.mdx (hunk 1 above)
Comment (aagonzales) [reply src/pages/contributing/get-started.mdx:69]: @jeanservaas See alison's comment here. Does this need an update?

**Review by alisonjoseph — APPROVED** [review]

**Pull request merged**